src/App.tsx

```diff
@@ -1,17 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import users from './api/users';
+import todos from './api/todos';
 import './App.css';
 
-import users from './api/users';
+const preparedTodos = [...todos].map((todo) => {
+  const newUser = users.find(user => user.id === todo.userId) || null;
+
+  if (newUser === null) {
+    throw new Error('choose someone');
+  }
+
+  return {
+    ...todo,
+    person: newUser,
+  };
+});
 
 const App: React.FC = () => {
+  const [name, setName] = useState('');
+  const [todo, setTodo] = useState('');
+  const [allTodos, setTodos] = useState(preparedTodos);
+
+  const addNewTodo = () => {
+    let selectedUser = users.find(user => name.includes(user.name)) || null;
+
+    if (name === '' || todo === '') {
+      // eslint-disable-next-line no-alert
+      alert(todo.length ? 'Please, choose a user' : 'Please, add text in field "What to do"');
```

Comment:
if not show an error message next to the `select` (`Please choose a user`)
if the `title` is empty show an error message next to the `title` field (`Please enter the title`)